### PR TITLE
[ENG-58] Outperforming & Profitable Months

### PR DESCRIPTION
**Pull request by sheridavandenbent**
# Results of merging this
Adds the metric Outperforming vs Market Months, and Profitable Months. Also bugfixes the outperforming & profitable weeks. Adds a new table in order to give these metrics some space, and fancy colours.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
   "max-open-trades": 3,
   "exposure-per-trade": 100,
   "starting-capital": 1000,
-  "backtesting-from": "2021-01-01",
-  "backtesting-to": "2021-07-01",
+  "backtesting-from": "2021-07-01",
+  "backtesting-to": "2021-07-02",
```
Comment (sheridavandenbent) [inline config.json:8]:
```suggestion
  "backtesting-from": "2021-01-01",
  "backtesting-to": "2021-07-01",
```

File: config.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "60": 4,
     "120": 3
   },
-  "pairs": ["BTC", "LTC", "ETH"],
+  "pairs": ["BTC"],
```
Comment (sheridavandenbent) [inline config.json:17]:
```suggestion
  "pairs": ["BTC", "LTC", "ETH"],
```

File: modules/output/results.py [hunk 3]
```diff
@@ -306,6 +340,22 @@ def create_coin_performance_table(justification: JustifyMethod, currency_symbol:
         coin_performance_table.add_column(f"Actual ({currency_symbol})", justify=justification)
         return coin_performance_table
 
+    @staticmethod
+    def create_coin_perf_per_timeframe_table(justification: JustifyMethod, currency_symbol: str) -> Table:
+        coin_perf_per_timeframe_table = Table(title="Coin Performance per Timeframe",
```
Comment (WillemGerritzen) [inline modules/output/results.py:345]: Would it make more sense to have the coin at the end of the title? Something like 'Weekly and Monthly Performance per Coin'

File: resources/setup/strategies/my_strategy.py [hunk 4]
```diff
@@ -62,7 +62,6 @@ def sell_signal(self, dataframe: DataFrame) -> DataFrame:
 
         dataframe.loc[
             (
```
Comment (WillemGerritzen) [inline resources/setup/strategies/my_strategy.py:64]: Did you remove the RSI > 70 on purpose?

**Review by WillemGerritzen — CHANGES_REQUESTED** [review]

File: modules/output/results.py (hunk 3 above)
Comment (sheridavandenbent) [reply modules/output/results.py:345]: Yes! Good idea. Renaming it.

File: resources/setup/strategies/my_strategy.py (hunk 4 above)
Comment (sheridavandenbent) [reply resources/setup/strategies/my_strategy.py:64]: Nope. I'll remove it. Thanks!

File: resources/setup/strategies/my_strategy.py [hunk 5]
```diff
@@ -62,7 +62,6 @@ def sell_signal(self, dataframe: DataFrame) -> DataFrame:
...
-                (dataframe['rsi'] > 70) &
                 (dataframe['volume'] > 0)
```
Comment (sheridavandenbent) [inline resources/setup/strategies/my_strategy.py:65]:
```suggestion
                (dataframe['rsi'] > 70) &
                (dataframe['volume'] > 0)
```

**Pull request merged**